sportsdataverse/nhl/nhl_api_web_parsers.py

```
@@ -163,7 +159,8 @@ def parse_nhl_web_landing(
 
 
 def parse_nhl_web_right_rail(
-    payload: Dict, section: str = None,
+    payload: Dict,
+    section: str = None,
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)